src/Tagger.Git/Tagger.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Omex.Log.Tagger.Git
+{
+	public static class Tagger
+	{
+		private static ITagger s_tagger;
```

Comment:
>s_tagger [](start = 25, length = 8)

would Lazy be more appropriate here? Will that called be used in a multithreaded environment?